[Avatar] Add shape parameter (#6103)
Shopify/polaris: polaris-react/src/components/Avatar/README.md

}
```

### Square avatar

> Use to present a square shaped for a merchant, customer, or business.

Reviewer: Is this the shape we want avatars to be going forward when representing all merchants (and their staff?) once unified Admin launches? Or is this just for Plus merchants?

```suggestion
Use a `square` `shape` when the avatar represents the merchant.
```
